ewah/hooks/plentymarkets.py

```diff
@@ -220,23 +222,37 @@ def get_data_in_batches(
         resource = self.format_resource(resource)
         url = self.endpoint + resource
 
-        if data_from:
-            if request_method == "get":
-                params[self._INCREMENTAL_FIELDS[resource][0]] = data_from.isoformat()
+        if resource == "/rest/orders/search":
+            if data_from and data_until:
+                params[
+                    "updatedAt"
+                ] = f"between:{data_from.isoformat()},{data_until.isoformat()}"
+            elif data_from:
+                params["updatedAt"] = f"gte:{data_from.isoformat()}"
+            elif data_until:
+                params["updatedAt"] = f"lte:{data_until.isoformat()}"
```

Comment:
Comment (non-blocking): Plentymarkets really cannot commit to one style, right? Feels like they've got all possible variations covered by now

Reply:
😿